webpack.config.js

```diff
@@ -164,14 +178,14 @@ module.exports = (env, argv) => {
                             options: {
                                 importLoaders: 1,
                                 sourceMap: true,
-                            }
+                            },
                         },
                         {
                             loader: 'postcss-loader',
                             ident: 'postcss',
                             options: {
-                                sourceMap: true,
-                                plugins: () => [
+                                "sourceMap": true,
+                                "plugins": () => [
```

Comment:
ideally we don't use quotes for props here - it's inconsistent and gives a different connotation to the object. Likewise for below

Reply:
Same same